src/stats/violin.jl

```diff
@@ -71,6 +73,7 @@ function plot!(plot::Violin)
                 npoints = n,
                 (bound === automatic ? NamedTuple() : (boundary = bound,))...,
                 (bw === automatic ? NamedTuple() : (bandwidth = bw,))...,
+                (w === automatic ? NamedTuple() : (weights = StatsBase.weights(view(w, idxs)),))...
```

Comment:
We should probably add a utility function to filter out `automatic` keywords, but that should be done separately.